usaspending_api/broker/management/commands/update_transactions.py

```diff
@@ -6,13 +6,15 @@
 from django.core.management.base import BaseCommand
 from django.db import connections, transaction as db_transaction, IntegrityError
 
-from usaspending_api.etl.broker_etl_helpers import dictfetchall
-from usaspending_api.awards.models import TransactionNormalized, TransactionFABS, TransactionFPDS
+from usaspending_api.agencies.models import Agency
+from usaspending_api.agencies.models import Agency, SubtierAgency, ToptierAgency
 from usaspending_api.awards.models import Award
+from usaspending_api.awards.models import TransactionNormalized, TransactionFABS, TransactionFPDS
```

Comment:
Award can be added to this import statement since they are all coming from `usaspending_api.awards.models`.